oauth2_http/java/com/google/auth/oauth2/ServiceAccountJwtAccessCredentials.java

```diff
@@ -399,6 +437,11 @@ public final String getPrivateKeyId() {
     return privateKeyId;
   }
 
+  @Override
+  public final String getUniverseDomain() {
+    return universeDomain;
+  }
```

Comment:
Can you add javadocs for this public method

Reply:
Done, thank you.